sdk/keyvault/keyvault-keys/package.json

```diff
@@ -64,9 +65,10 @@
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
     "build:minify": "uglifyjs -c -m --comments --source-map \"content='./dist/index.js.map'\" -o ./dist/index.min.js ./dist/index.js 2>&1",
     "build:test": "npm run build",
-    "build": "tsc && rollup -c rollup.config.js 2>&1",
+    "build": "tsc && rollup -c rollup.config.js 2>&1 && npm run extract-api",
```

Comment:
In this model, `tsc` gets run twice.
If I recall correctly, the api extractor doesnt care for rollup. Therefore, since the `extract-api` script already runs tsc, can we update this to
`"build": npm run extract-api && rollup -c rollup.config.js 2>&1` This way `tsc` runs just once.